app/components/ProductDescription.jsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from "react";
+import { useCartProducts } from "../Domain/cartContext";
+import { useRouter } from "next/navigation";
+
+export default function ProductDescription({product ,title, description, quantity, price,setShowNotification }) {
+  const { addToCart } = useCartProducts();
+  const [selectedQuantity, setSelectedQuantity] = useState(0);
+  const router = useRouter();
+
+  const quantityOptions = [
+    <option key={0} value={0}>
```

Comment:
The value also needs to be set to 1 so adding to cart by default adds 1 instead of 0